### ISSUE TITLE
NTR, pulmo: Panacinar emphysema

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5577]
**Preferred term label:**
Panacinar emphysema

**Synonyms**
Panlobular emphysema

**Definition (free text, please give PubMed ID)**
Panacinar emphysema involves all portions of the acinus and secondary pulmonary lobule more or less uniformly. It predominates in the lower lobes and is the form of emphysema associated with1-antitrypsin deficiency. CT scans show a generalized decrease of the lung parenchyma with a decrease in the caliber of blood vessels in the affected lung. Severe panacinar emphysema may coexist and merge with severe centrilobular emphysema. The appearance of feature less decreased attenuation may be indistinguishable from severe constrictive obliterative bronchiolitis. 

PMID: 18195376, PMID: 8370825, https://doi.org/10.1016/S0009-9260(05)80011-X

**Parent term (use hpo.jax.org/app)**
Emphysema HP:0002097

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 OMIM 613490, ALPHA-1-ANTITRYPSIN DEFICIENCY
ORPHA:1303, Bronchiolitis obliterans with obstructive pulmonary disease

**Your nano-attribution (ORCID)**
LMU crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
Adding term as HP:0032967. Added the proposed annotations to our annotation tracker.
